core/lib/zksync_core/src/api_server/web3/tests/filters.rs

```diff
@@ -263,3 +265,42 @@ impl HttpTest for LogFilterChangesWithBlockBoundariesTest {
 async fn log_filter_changes_with_block_boundaries() {
     test_http_server(LogFilterChangesWithBlockBoundariesTest).await;
 }
+
+#[derive(Debug)]
+struct DisableFiltersTest;
+
+#[async_trait]
+impl HttpTest for DisableFiltersTest {
+    async fn test(&self, client: &HttpClient, _pool: &ConnectionPool) -> anyhow::Result<()> {
+        use jsonrpsee::{core::client::Error, types::error::ErrorCode};
+
+        fn assert_not_implemented<T: Debug>(result: Result<T, Error>) {
```

Comment:
Nit: I'd suggest to move this function to the module level; it'd be easier to read, and it could be later reused in other tests.

Reply:
Hmm, my reasoning behind making this scoped was that it uses collision-heavy types (`Error` and `ErrorCode`) but fair enough

On this point, do we have a styling guide perhaps? Just so I can get aligned on these things with everyone else